lib/timeline/Timeline.js

```diff
@@ -301,12 +301,27 @@ Timeline.prototype.setData = function (data) {
  *                                    Default duration is 500 ms, and default easing
  *                                    function is 'easeInOutQuad'.
  *                                    Only applicable when option focus is true.
+ *                                `selectEvent: Event`
+ *                                    Call the select event manually to make sure the selected items
+ *                                    will be highlighted in the timeline. Specific use for JQuery-
+ *                                    drag&drop of timelime items when using IE11.
  */
 Timeline.prototype.setSelection = function(ids, options) {
-  this.itemSet && this.itemSet.setSelection(ids);
+  var oldSelection = [];
+  this.itemSet && (oldSelection = this.itemSet.getSelection()) && this.itemSet.setSelection(ids);
 
   if (options && options.focus) {
     this.focus(ids, options);
+  } else if (options && options.selectEvent) {
```

Comment:
Missing documentation in `docs/timeline/index.html`. You should mention this option in the docs to allow people to know of it's existence. 

I also don't love the option name. I kind of like `options.emitSelectEvent` better